openslides/assignments/static/js/assignments/site.js

```diff
@@ -89,6 +92,9 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
                                         return Assignment.loadRelations(assignment, 'agenda_item');
```

Comment:
please try to remove the loadRelations line and test if you can show the agend item number in title of election (should be work because you load all items later)